### PR TITLE
feat: slider add support for disable keyboard events

### PR DESCRIPTION
**Pull request by yoyo837**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
#21695
https://github.com/react-component/slider/pull/852

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    slider add support for disable keyboard events       |
| 🇨🇳 Chinese |     slider 添加 `keyboard` 属性以支持禁用键盘事件      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
上游PR已经合并老久了, 应该合上来.

**Review by li-jia-nan — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -140,7 +140,7 @@
     "rc-resize-observer": "^1.2.0",
     "rc-segmented": "~2.1.0",
     "rc-select": "~14.2.0",
-    "rc-slider": "~10.0.0",
+    "rc-slider": "~10.1.0",
```
Comment (afc163) [inline package.json:143]: 文档和 ts 需要更新一下？

Comment (yoyo837) [reply package.json:143]: 忘记了

**Comment by yoyo837** [comment]
@SocketSecurity ignore @swc/core@1.3.32
@SocketSecurity ignore puppeteer@19.6.3

**Review by afc163 — APPROVED** [review]

**Pull request merged**